### PR TITLE
svg and image added, not scaling correctly

### PR DESCRIPTION
**Pull request by kapil-krishna**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+declare module '*.jpg';
```
Comment (MichaelWalker) [inline src/index.d.ts]: I don't think we need this any more - could you remove it?

File: src/map.tsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import './style/map.scss';
+
+// import mapImage from './mars-map-desktop-res.jpg';
+
+// function MarsMap() {
+//     <picture>
+//         <source media="(min-width: 768px)" srcset={mapImage}>
+
+//     </picture>
+// }
```
Comment (MichaelWalker) [inline src/map.tsx]: Could you also kill all of the commented out code?

File: src/map.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+
+
+export function MarsMapImg() {
+    const image = require('./mars-map-desktop-res.jpg');
+
+   return <svg>
+       <image href={image}></image>
+   {/* <circle cx="50" cy="50" r="40" stroke="green" stroke-width="4" fill="yellow" /> */}
```
Comment (MichaelWalker) [inline src/map.tsx]: also as we discussed earlier, we have a better way to do this now! :)

**Comment by JakeHorton94** [comment]
Nah we don't need this anymore, how do I get rid of it?

**Pull request closed without merging**